frontend/src/styles/globals.scss

```diff
@@ -35,12 +34,23 @@ body,
   // empty space at the bottom of the page, below the footer.
   // Since the user can scroll in `.non-header-wrapper`, everything outside that
   // that still overflows these elements can safely be hidden:
-  overflow: hidden;
+  // overflow: hidden;
```

Comment:
I'm not sure if it was necessary to remove this, but if it was, might as well delete the full line and the comment above it. (And ideally also an explanation of why it needed to be removed :) )